crates/config/src/soldeer.rs

```diff
@@ -11,7 +11,7 @@ pub struct SoldeerDependency {
     pub version: String,
 
     /// The url from where the dependency was retrieved
-    pub url: String,
+    pub url: Option<String>,
```

Comment:
should this be #[serde(default, skip_serializing_if = "Option::is_none"]

Reply:
added